docs/source/scribe_data/cli.rst

```diff
@@ -1,6 +1,232 @@
+4. ``convert`` (alias: ``c``)
+
+List Command
+~~~~~~~~~~~~
+
+Description: List languages, data types and combinations of each that Scribe-Data can be used for.
+
+Usage:
+
+.. code-block:: bash
+
+   scribe-data list [options]
+
+Options:
+^^^^^^^^
+
+- ``-lang, --language [LANGUAGE]``: List options for all or given languages.
+- ``-dt, --data-type [DATA_TYPE]``: List options for all or given data types.
+- ``-a, --all ALL``: List all languages and data types.
+
+Example output:
+
+.. code-block:: text
+
+   $ scribe-data list
+   Language     ISO  QID
+   -----------------------
+   English      en   Q1860
+   French       fr   Q150
+   German       de   Q188
+   Italian      it   Q652
+   Portuguese   pt   Q5146
+   Russian      ru   Q7737
+   Spanish      es   Q1321
+   Swedish      sv   Q9027
+   -----------------------
+   Available data types: All languages
+   -----------------------------------
+   nouns
+   prepositions
+   translations
+   verbs
+   -----------------------------------
+
+Query Command
+~~~~~~~~~~~~~
+
+Description: Query data from Wikidata for the given languages and data types.
+
+Usage:
+
+.. code-block:: bash
+
+   scribe-data query [options]
+
+Options:
+^^^^^^^^
+
+- ``-lang, --language LANGUAGE``: The language(s) to query.
+- ``-dt, --data-type DATA_TYPE``: The data type(s) to query.
+- ``-od, --output-dir OUTPUT_DIR``: The output directory path for results.
+- ``-ot, --output-type {json,csv,tsv}``: The output file type.
+- ``-o, --overwrite``: Whether to overwrite existing files (default: False).
+- ``-a, --all ALL``: Query all languages and data types.
+- ``-i, --interactive``: Run in interactive mode.
+
+Example:
+
+.. code-block:: bash
+
+   $ scribe-data query -l English --data-type verbs -od ~/Desktop/gsoc/wiki/Scribe-Data
+
+Behavior and Output:
+^^^^^^^^^^^^^^^^^^^^
+
+1. The command will first check for existing data:
+
+   .. code-block:: text
+
+      Updating data for language: English, data type: ['verbs']
+      Data updated:   0%|
+
+2. If existing files are found, you'll be prompted to choose an option:
+
+   .. code-block:: text
+
+      Existing file(s) found for English verbs:
+      1. verbs_2024_07_07_00_28_44.json
+      2. verbs_2024_07_07_00_29_20.json
+      Choose an option:
+      1. Keep existing (skip update)
+      2. Overwrite existing
+      3. Keep both
+      4. Cancel
+      Enter your choice (1-4):
+
+3. After making a selection, the querying process begins:
+
+   .. code-block:: text
+
+      Querying and formatting English verbs
+      Data updated: 100%|████████████████████████| 1/1 [00:29<00:00, 29.73s/process]
+
+4. If no data is found, you'll see a warning:
+
+   .. code-block:: text
+
+      No data found for language 'english' and data type '['verbs']'.
+      Warning: No data file found for 'English' ['verbs']
+
+Notes:
+^^^^^^
+
+1. The command uses ``-l`` for language instead of ``-lang`` as previously documented. Both forms appear to be accepted.
```

Comment:
I don't think we need to draw a distinction for this as this is the original release of it :) Not too many people have seen the original.